data/data/bootstrap/files/etc/containers/registries.conf.template

```diff
@@ -0,0 +1,13 @@
+{{ range $r := .Registries -}}
```

Comment:
Couldn’t this just encode a `sysregistriesv2.V2RegistriesConf` object directly instead of hard-coding the current set of fields of `$r`?

Reply:
… sorry, probably not trivially in the template language, without adding helpers at least.